docs/rfcs/1002-merging-process.md

```diff
@@ -0,0 +1,383 @@
+  pull request is recommended.
+
+### The pull request checklist
+
+The checklist is displayed before submitting a pull request:
+
+```markdown
+<!-- You can erase any parts of this template not applicable to your pull request -->
+
+## Describe your changes
+
+<!-- Why this PR exist ? what is its goal ?
+     Give some context about the changes
+     Draw attention to the details that should not be overlooked -->
+
+## Checklist
+
+Before you submit this pull request, please make sure to:
+
+- [ ] Keep changes in the pull request as small as possible
+   <!-- Move unrelated changes to a new pull request -->
+- [ ] Ensure the commit history is sanitized
+   <!-- Commits should have a meaningful title and contain a coherent set of changes
+        Squash commits that are only relevant to the branch context -->
+- [ ] Give a meaningful title to your PR
+- [ ] Describe your changes
+   <!-- Give some context about the changes
+        Draw attention to the details that should not be overlooked -->
+- [ ] Link any related issue in the description
+   <!-- You can add `closes #<IssueID>` to close the issue when the PR is merged -->
+- [ ] Link any dependent pull request in the description
+
+If this PR adds a new feature or fixes a bug:
+
+- [ ] I have added or updated relevant tests
+
+If this PR affects the user:
+
+<!-- Changes that have an impact for the user:
+     - UI has been changed
+     - A message displayed to the user has been changed
+     - A new feature has been added
+     - A use case has been modified
+     - ... -->
+
+- [ ] I have created a news fragment
+   <!-- The news fragments are used to generate the changelog -->
+- [ ] I have updated the user documentation
+```
+
+## Reviewing a pull request
+
+Before a pull request can be merged, it need to be reviewed to meet our quality
+standards.
+
+The review consists in two steps:
+
+- [The automated review](#the-automated-review)
+- [The peer review](#the-peer-review)
+
+Also, there are some extra steps for [specific pull requests reviews](#specific-pull-request-reviews).
+
+### The automated review
+
+The goal is to automate the systematic and tedious checks that can be easily
+forgotten by a peer.
+
+This step is responsible for:
+
+- Ensuring codebase compiles (or equivalent)
+- Executing tests and verifying their result (cargo test, pytest, ...)
+- Verifying coding style using the configured linters (rustfmt, clippy, ruff,
+  eslint, ...)
+- Checking spell issues (cspell)
+- Ensuring the app can still be packaged with the proposed change
+- Ensuring commit signatures are valid (gpg signed commit)
+- Verifying simple checks from the [Pull-request checklist](#the-pull-request-checklist)
+- Verifying simple check on commit titles
+
+### The peer review
+
+The goal is not only to ensure quality standards missed in the automated review,
+but also to encourage **team collaboration** and **knowledge sharing**.
```

Comment:
and application security (A single developer cannot include backdoors without being seen).

Reply:
```suggestion
but also to encourage **team collaboration**, **application security** and **knowledge sharing**.
```